lib/capsule_crm/serializer.rb

```diff
@@ -84,7 +84,7 @@ def attributes
         attrs.each do |key, value|
           attrs[key] = value.to_s(:db) if value.is_a?(Date)
           if value.is_a?(DateTime)
-            attrs[key] = value.strftime("%Y-%m-%dT%H:%M:%SZ")
+            attrs[key] = value.utc.strftime("%Y-%m-%dT%H:%M:%SZ")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.